src/modules/getOuLevelAndGroupText.js

```diff
@@ -61,12 +65,26 @@ const getLevelAndGroupText = (items, metaData, isLevel) => {
         .join(', ')
 
     if (!staticOuNames) {
-        return `${allDynamicOuNames} ${
-            isLevel ? i18n.t('levels') : i18n.t('groups')
-        }`
+        if (isLevel) {
+            return i18n.t('{{allDynamicOuNames}} levels', {
+                allDynamicOuNames,
+            })
+        }
+
+        return i18n.t('{{allDynamicOuNames}} groups', {
+            allDynamicOuNames,
+        })
+    }
+
+    if (isLevel) {
+        return i18n.t('{{allDynamicOuNames}} levels in {{staticOuNames}}', {
```

Comment:
Great update with the interpolations. Could we possibly use variable names that are easier to understand for the translators?